### PR TITLE
New action that can be run on cron to merge auto merge PRs

### PR DESCRIPTION
**Pull request by ASupinski**
since we have a bunch of repositories where the circleCI build can block the merge of a branch and until the github feature request to get an event to key off of when the PR becomes mergable I have this action that can be run on cron to try and merge every pull request that has a specified label.

### CODE REVIEW

File: automerge/entrypoint.sh [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+PRs="$(hub pr list -f "%L||%U||%B%n")"
+
+for pr in PRs do
+  read -ra Parts <<< "$(echo pr | tr "||" "\n")"
+  label="$Parts[0]"
+  url="$Parts[1]"
+  base="$Parts[2]"
+
+  #case insensitive compare by using the ,, to lower case
+  if [ "${label,,}" = "${MERGE_LABEL,,}"]
```
Comment (xsalazar) [inline automerge/entrypoint.sh]: Missing space before final `]` in `... = "${MERGE_LABEL,,}"]`

Comment (ASupinski) [reply automerge/entrypoint.sh]: fixed, thx!

File: automerge/entrypoint.sh [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+        echo "DEBUG: successfully pushed ${GITHUB_REF} merged into $1" || \
+        echo "DEBUG: pushing ${GITHUB_REF} merged into $1 failed"
+
+    )
+}
+
+#list all open pull requests with labels||url||baseBranch
+PRs="$(hub pr list -f "%L||%U||%B%n")"
+
+for pr in PRs do
```
Comment (xsalazar) [inline automerge/entrypoint.sh]: - Missing `$` on list `for pr in $PRs`
- Should also have semicolon:
   ```
   for pr in $PRs; do
   ```
   or newline:
   ```
   for pr in $PRs
   do
   ```

File: automerge/entrypoint.sh [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+  read -ra Parts <<< "$(echo pr | tr "||" "\n")"
```
Comment (xsalazar) [inline automerge/entrypoint.sh]: Should be `$pr` for accessor in `"$(echo $pr | tr "||" "\n")"`

File: automerge/entrypoint.sh [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+  label="$Parts[0]"
+  url="$Parts[1]"
+  base="$Parts[2]"
```
Comment (xsalazar) [inline automerge/entrypoint.sh]: Expand arrays with `{}`, i.e. `label="${Parts[0]}"

File: automerge/entrypoint.sh (hunk 2 above)
Comment (ASupinski) [reply automerge/entrypoint.sh]: done

File: automerge/entrypoint.sh (hunk 3 above)
Comment (ASupinski) [reply automerge/entrypoint.sh]: done

File: automerge/entrypoint.sh (hunk 4 above)
Comment (ASupinski) [reply automerge/entrypoint.sh]: done

File: automerge/entrypoint.sh (hunk 2 above)
Comment (ASupinski) [reply automerge/entrypoint.sh]: doing the $ on the itterator name fails linting

**Pull request merged**